### ISSUE TITLE
Add Google Analytics widget

### ISSUE THREAD

**Issue by KeizerDev** [dan-silver/cast-dashboard-android-app#2]
This would really be awesome if it was there!

**Comment by dan-silver** [comment]
Good idea! I'm currently working on more Google widgets like Google calendar, youtube (highly requested), gmail, etc. Since users now log in with the google accounts to the app, adding a google analytics widget is definitely possible and I'll keep this issue open as a reminder!

**Comment by KeizerDev** [comment]
@dan-silver Cool, looking for a good Chromecast dashboard where I can see multiple instances of gAnalytics with all realtime data

**Comment by anykytenko** [comment]
This is https://play.google.com/store/apps/details?id=org.nikas.analytics.widget a really good widget for Android desktop. It is free and has all you need

**Comment by KeizerDev** [comment]
Also support for realtime stats?
